src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/OverloadOperatorEqualsOnOverridingValueTypeEqualsTests.Fixer.cs

```diff
@@ -82,5 +82,123 @@ End Operator
 End Structure
 ");
         }
+
+        [Fact]
+        public async Task CA2231_CSharp_FixAll()
+        {
+            await new VerifyCS.Test
+            {
+                TestCode = @"
+using System;
+
+public struct [|A|]
+{
+    public override bool Equals(object obj)
+    {
+        return true;
+    }
+}
+
+public struct [|B|]
+{
+    public override bool Equals(object obj)
+    {
+        return true;
+    }
+}",
+                FixedCode = @"
+using System;
+
+public struct A
+{
+    public override bool Equals(object obj)
+    {
+        return true;
+    }
+
+    public static bool operator ==(A left, A right)
+    {
+        return left.Equals(right);
+    }
+
+    public static bool operator !=(A left, A right)
+    {
+        return !(left == right);
+    }
+}
+
+public struct B
+{
+    public override bool Equals(object obj)
+    {
+        return true;
+    }
+
+    public static bool operator ==(B left, B right)
+    {
+        return left.Equals(right);
+    }
+
+    public static bool operator !=(B left, B right)
+    {
+        return !(left == right);
+    }
+}",
+                NumberOfFixAllIterations = 1
```

Comment:
📝 Fix All is always tested, and `1` is the default value for this property. Therefore:

* The name of this test is a bit redundant. It would be better to call it `CA2231_CSharp_MultipleViolations`
* The method can use the simpler form `await VerifyCS.VerifyCodeFixAsync`

Reply:
Thank you for the clarification! I'll create a cleanup PR.